pipeline/not_value.go

```diff
@@ -0,0 +1,13 @@
+package pipeline
+
+// NotAValue - value for check setuping in fields
```

Comment:
If you are using such a word there must be double `setupping`. But anyway there is no such a word, `setting up` fits better.

Reply:
++